evm/utils/env.py

```diff
@@ -0,0 +1,256 @@
+    values. If not present in the environment and no default is specified,
+    ``None`` is returned.
+
+    :param name: The name of the environment variable be pulled
+    :type name: str
+
+    :param truthy_values: An iterable of values that should be considered
+    truthy.
+    :type truthy_values: iterable
+
+    :param required: Whether the environment variable is required. If ``True``
+    and the variable is not present, a ``KeyError`` is raised.
+    :type required: bool
+
+    :param default: The value to return if the environment variable is not
+    present. (Providing a default alongside setting ``required=True`` will raise
+    a ``ValueError``)
+    :type default: bool
+    """
+    value = get_env_value(name, required=required, default=default)
+    if value is empty:
+        return None
+    return value in TRUE_VALUES
+
+
+def env_string(name, required=False, default=empty):
+    """Pulls an environment variable out of the environment returning it as a
+    string. If not present in the environment and no default is specified, an
+    empty string is returned.
+
+    :param name: The name of the environment variable be pulled
+    :type name: str
+
+    :param required: Whether the environment variable is required. If ``True``
+    and the variable is not present, a ``KeyError`` is raised.
+    :type required: bool
+
+    :param default: The value to return if the environment variable is not
+    present. (Providing a default alongside setting ``required=True`` will raise
+    a ``ValueError``)
+    :type default: bool
+    """
+    value = get_env_value(name, default=default, required=required)
+    if value is empty:
+        value = ''
+    return value
+
+
+def env_list(name, separator=',', required=False, default=empty):
+    """Pulls an environment variable out of the environment, splitting it on a
+    separator, and returning it as a list. Extra whitespace on the list values
+    is stripped. List values that evaluate as falsy are removed. If not present
+    and no default specified, an empty list is returned.
+
+    :param name: The name of the environment variable be pulled
+    :type name: str
+
+    :param separator: The separator that the string should be split on.
+    :type separator: str
+
+    :param required: Whether the environment variable is required. If ``True``
+    and the variable is not present, a ``KeyError`` is raised.
+    :type required: bool
+
+    :param default: The value to return if the environment variable is not
+    present. (Providing a default alongside setting ``required=True`` will raise
+    a ``ValueError``)
+    :type default: bool
+    """
+    value = get_env_value(name, required=required, default=default)
+    if value is empty:
+        return []
+    # wrapped in list to force evaluation in python 3
+    return list(filter(bool, [v.strip() for v in value.split(separator)]))
+
+
+def env_timestamp(name, required=False, default=empty):
```

Comment:
I glossed over this on the first one not realizing it returned `datetime.datetime` objects.  I think we could remove it as well for the same reasons as `iso8601`

Reply:
No problem.:)